lib/ts/ink_align.h

```diff
@@ -53,6 +53,12 @@ union Alias64 {
 /** Default alignment */
 #define INK_ALIGN_DEFAULT(size) INK_ALIGN(size, INK_MIN_ALIGN)
 
+static inline size_t
+aligned_spacing(size_t len, size_t block = INK_MIN_ALIGN)
+{
+  return INK_ALIGN(len, block);
+}
```

Comment:
The semantic burden is much *worse* with INK_ALIGN.   It does no aligning of any pointer.   The burden is *worse* also because INK_ALIGN is the odd man out.  All the other functions in that header look like the ones below.

My case is that INK_ALIGN does not do what the name is... and even has the "INK_*" prefix that seems rather redundant and old compared with ats_